src/style.css

```diff
@@ -13,102 +15,185 @@ html {
 }
 
 body {
-  background-color: rgb(210, 186, 186);
+  width: 100%;
   display: flex;
   justify-content: center;
-  align-items: center;
-  width: 100%;
-  height: 100vh;
-  margin: 0;
-  padding: 0;
 }
 
-.todo-list-container {
-  list-style: none;
-  display: flex;
-  flex-direction: column;
-  background-color: rgb(244, 238, 238);
-  width: 50%;
-  min-width: 30rem;
-  height: auto;
-  padding: 20px 0 0 0;
-  box-shadow: 0 4px 8px 0 rgba(31, 30, 30, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-  border: 1px solid rgba(59, 57, 57, 0.18);
-  border-radius: 0.5rem;
+.wrapper {
+  width: 50rem;
+  margin-left: 7rem;
+  margin-top: 90px;
+  border-radius: 10px;
+  font-family: "Yantramanav", sans-serif;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
 }
 
-.tasks {
+header {
   display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-size: 30px;
+  font-weight: 600;
+  height: 60px;
   align-items: center;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.listElements {
+  display: flex;
   justify-content: space-between;
-  border-bottom: 1px solid #5c5c5c;
-  padding: 1rem 2rem;
+  cursor: pointer;
 }
 
-.main-header {
-  margin: 0;
+.fas {
+  color: rgb(139, 137, 137);
 }
 
-.add-item {
-  background: none;
-  outline: none;
-  font-family: "Roboto", Arial, Helvetica, sans-serif;
-  width: 100%;
-  height: 4rem;
+.trash {
+  display: none;
+}
+
+.no-outline {
   border: none;
+  outline-width: 0;
+}
+
+header span {
+  font-size: 30px;
+  font-weight: 300;
+  cursor: pointer;
 }
 
-.add-item::placeholder {
+.inputField {
+  display: flex;
+  width: 100%;
+  height: 60px;
+}
+
+input {
+  background-color: transparent;
+  border: 0 solid;
+}
+
+.inputField #input-box {
+  height: 100%;
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  padding-left: 10px;
+}
+
+.inputField #input-box::placeholder {
   font-style: italic;
-  letter-spacing: 0.1rem;
 }
 
-.enter-button {
-  transform: rotate(90deg);
-  border: none;
-  background: none;
+.inputField span {
+  display: flex;
+  align-items: center;
+  margin-right: 10px;
+  cursor: pointer;
 }
 
-.checkbox {
-  width: 2rem;
-  height: 2rem;
+.d-flex {
+  display: flex;
 }
 
-.description {
-  margin: 0;
-  padding: 1.6rem 2.1rem 0 2rem;
+.center-items {
+  align-items: center;
+}
+
+.list-group-1 {
   width: 100%;
-  resize: none;
-  background: none;
-  border: none;
+  height: 60%;
 }
 
-.enter-button,
-.task-button,
-.clear-tasks {
-  padding: 0.5rem;
-  font-size: 1.5rem;
-  border: none;
-  color: #414141;
-  background: none;
+.task-description {
+  margin-left: 10px;
```

Comment:
- [ ] For a better user interface please 🙏🏽 increase the text size and give it contrast otherwise the activities are imperceptible.

![image](https://user-images.githubusercontent.com/58642949/196009803-b710325b-022e-4b98-acd0-d65e23e78ec0.png)